Documentation fixes (#2342)
tabler/tabler: docs/content/emails/introduction/contents.md

This file contains the license information for the Tabler Emails package.

### 4. Readme: `readme.html`

> This file with the main information about the Tabler Emails package. It contains a brief description of the package and instructions on how to use it.

Reviewer: How about: 
```suggestion
This file contains the main information about the Tabler Emails package, including a brief description and instructions on how to use it.
```
